docs/uc.md

```diff
@@ -0,0 +1,60 @@
+# Using dbt-databricks with Unity Catalog
+
+[Unity Catalog](https://www.databricks.com/product/unity-catalog) provides a SQL interface to manage tables, views and more across Databricks workspaces using standard SQL. dbt-databricks supports 
+
+# Benefits of Unity Catalog for analytics engineers
+
+We strongly recommend using Unity Catalog for all dbt projects on Databricks. Unity Catalog brings two important benefits to analytics engineers:
+
+1. Unity Catalog offers a three-level namespace: catalog > schema > table/view. Note that schema is a synonym for database in earlier releases of dbt-databricks. This lets you manage and isolate data and avoids polluting a single database with unrelated tables and views. For example, you can data between your finance and HR teams inside a production catalog called `prod` in two schemas, `finance` and `hr`:
+
+```
+catalog: prod
+    │
+    │schema: finance
+    ├──────┐
+    │      ├───── revenue
+    │      └───── chargebacks
+    │
+    │schema: hr
+    └──────┐
+           ├───── people
+           └───── salaries
+```
+
+2. All privilege management is through standard SQL, which means you don't have to learn a new method of managing access.
+3. Unity Catalog offers [automatic column-level lineage](https://docs.databricks.com/data-governance/unity-catalog/data-lineage.html).
+
+# Default catalog
+When you run `dbt init`, the CLI will ask you for a catalog. This is the _default_ catalog. Note that dbt models can refer to tables and views in any catalog (as long as you have appropriate privileges). The default catalog is simply where dbt will create tables and views by default.
```

Comment:
> dbt models can refer to tables and views in any catalog

We might want to mention about `hive_metastore`. Cross-catalog queries between UC and hive metastore have limitations.

Reply:
@ueshin PTAL